src/github.com/matrix-org/dendrite/syncapi/sync/request.go

```diff
@@ -26,7 +26,10 @@ import (
 )
 
 const defaultSyncTimeout = time.Duration(30) * time.Second
-const defaultTimelineLimit = 20
+
+// DefaultTimelineLimit is the default number limit of timeline entries if no
+// limit was given.
+const DefaultTimelineLimit = 20
```

Comment:
no particular objection to this, but it sees unnecessary given you haven't added any references to it outside the package?